lib/ts/recipe/multifactorauth/multiFactorAuthClaim.ts

```diff
@@ -143,7 +133,37 @@ export class MultiFactorAuthClaimClass extends SessionClaim<MFAClaimValue> {
         hasCompletedFactors(requirements: MFARequirementList, id?: string): SessionClaimValidator;
     };
 
-    public buildNextArray(completedClaims: MFAClaimValue["c"], requirements: MFARequirementList): string[] {
+    public isRequirementsSatisfied(completedClaims: MFAClaimValue["c"], requirements: MFARequirementList): boolean {
```

Comment:
```suggestion
    public isRequirementListSatisfied(completedClaims: MFAClaimValue["c"], requirementList: MFARequirementList): boolean {
```

Reply:
also, this partially duplicates the functionality of `getNextSetOfUnsatisfiedFactors`. I think we should de-duplicate this code (this is kind of the same as checking if the length of unsatisfied factors is 0)

Reply:
Done